packages/keystone/preprocessors/schemaDoc.js

```diff
@@ -58,11 +59,12 @@ const { GQL_CUSTOM_SCHEMA_TYPE } = require('../schema')
  * Formats error for documentation of GraphQL API in human readable way
  * @param {GQLError} errorConfig
  * @return {string}
- * TODO: localize error message
  */
-const formatError = (errorConfig) => (
-    '`' + JSON.stringify(errorConfig, null, 2) + '`'
-)
+const formatError = (errorConfig) => {
+    const error = new GQLError(errorConfig, {})  // render error with DEFAULT_LOCALE
```

Comment:
Default value can be any on a local project. 
But most likely the documentation should always be English

I wouldn't want to change the default value just to generate documentation correctly...

Reply:
It will also be unclear to third-party developers

Reply:
Yep. I'll change DEFAULT_LOCALE to 'en' in example